src/api/app/views/webui2/webui/project/_breadcrumb_items.html.haml

```diff
@@ -16,3 +16,6 @@
   - elsif current_page?(project_config_path(@project))
     %li.breadcrumb-item.active{ 'aria-current' => 'page' }
       Configuration
+  - elsif current_page?(project_meta_config_path(@project))
```

Comment:
Again, this should be `- elsif current_page?(project_meta_path(@project))` which is `project/meta/myproject`

Reply:
@dmarcoux ops. did it :)